python/test/unit/language/test_core_amd.py

```diff
@@ -1199,15 +1199,17 @@ def kernel(X, stride_xm, stride_xn,
 
 
 # MFMA Test Dot tests
-@pytest.mark.parametrize("M, N, K, num_warps, col_a, col_b, epilogue, allow_tf32, dtype",
-                         [(*shape, 2, False, False, epilogue, allow_tf32, dtype)
+@pytest.mark.parametrize("M, N, K, num_warps, col_a, col_b, epilogue, allow_tf32, in_dtype, out_dtype",
+                         [(*shape, 2, False, False, epilogue, allow_tf32, in_dtype, out_dtype)
                           for shape in [(64, 64, 64), (32, 32, 32)]
```

Comment:
Why are we not testing (16,16,16)? It is in upstream and when I try it I get a segfault.

Reply:
If you are running them on MI100/200 GPU, it tries to use MFMA instructions with minimal M/N size of 32.

Reply:
@micmelesse 
Upd. I've added workaround with FMA instructions, so 16x16x16 tests work